src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Used to represent all the Tasks
+ * to be done as it
+ * contains a list of all tasks
+ */
+public class TaskList {
+    private ArrayList<Task> lst;
+
+
+    TaskList(ArrayList<Task> lst) {
+        this.lst = lst;
+    }
+
+    TaskList() {
+        lst = new ArrayList<>();
+    }
+
+    /**
+     * Returns the task in specified index
+     * @param ind index of task
+     * @return Task at index ind
+     */
```

Comment:
I realise many of your JavaDoc share this coding style issue:
There needs to be a whitespace between the description and the @ statements. I think fullstops are also required.